pkg/api/v1/routes/routes.go

```diff
@@ -63,6 +66,22 @@ func WithConditionDefinitions(defs ptypes.ConditionDefinitions) Option {
 	}
 }
 
+// apiHandler is a function that performs real work for the ConditionOrc API
```

Comment:
I don't know that I love the `gin-metrics` package, because I think our use-case is simpler than what this package provides. I don't see a need for tracking byte counts of requests and replies, for example. I think being explicit in our usage of prometheus gives us more flexibility in extending and adapting our usage in the future.

Reply:
One thing I *do* like about that package though, is that he uses a middle-ware paradigm instead of a function-wrapping paradigm. That might be nice to examine as an example hook for Auth/Az purposes later, or perhaps to refactor this.

Reply:
yep, we do have a middleware that can we can use for Auth/Az later https://github.com/metal-toolbox/hollow-toolbox/tree/main/ginauth